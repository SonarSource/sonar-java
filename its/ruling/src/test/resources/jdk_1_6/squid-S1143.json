{
'jdk_1_6:java/io/ObjectInputStream.java':[
3094,
],
}
