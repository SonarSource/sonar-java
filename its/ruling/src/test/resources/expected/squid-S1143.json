{
'project:com/sun/org/apache/xerces/internal/xinclude/XIncludeHandler.java':[
1763,
],
'project:javax/swing/JEditorPane.java':[
681,
],
}
